Documentation/building/freebsd-instructions.md

```diff
@@ -22,11 +22,12 @@ Install the following packages for the toolchain:
 - llvm37 (includes LLVM 3.7, Clang 3.7 and LLDB 3.7)
 - libunwind
 - gettext
+- icu
 
 To install the packages you need:
 
 ```sh
-janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake libunwind gettext llvm37
+janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake libunwind gettext llvm37 icu
```

Comment:
I think everyone just followed my convention from when I wrote the original doc.  I like to make it super explicit about what's getting typed into the prompt, so it helps to make a prompt look like a prompt, IMHO.